src/tests/unit/audit_remediation/test_audit_remediation_base.py

```diff
@@ -478,3 +480,72 @@ def test_run_async_faceting_query(cbcsdk_mock):
     assert len(result) == 1
     assert result[0].field == 'fields.version'
     assert len(result[0].values) == 4
+
+
+def test_result_set_run_ids(cbcsdk_mock):
+    """Testing set_run_ids"""
+    api = cbcsdk_mock.api
+    query = api.select(Result).set_run_ids(["abcdefghijklmnopqrstuvwxyz123456", "fckjyssfusuuutlkpocky82luvnl0sol"])
+    assert query._criteria["run_id"] == ["abcdefghijklmnopqrstuvwxyz123456", "fckjyssfusuuutlkpocky82luvnl0sol"]
+
+
+def test_result_set_time_received(cbcsdk_mock):
+    """Testing set_time_received"""
+    api = cbcsdk_mock.api
+    query = api.select(Result).set_time_received(range="-3h")
+    assert query._criteria["time_received"] == {"range": "-3h"}
+
+    query.set_time_received(start="2023-12-10T00:00:00.000Z", end="2023-12-11T00:00:00.000Z")
+    assert query._criteria["time_received"] == {
+        "start": "2023-12-10T00:00:00.000Z",
+        "end": "2023-12-11T00:00:00.000Z"
+    }
+
+    with pytest.raises(ApiError):
+        query.set_time_received(start="2023-12-10T00:00:00.000Z", end="2023-12-11T00:00:00.000Z", range="-3h")
+
+
+def test_result_scroll(cbcsdk_mock):
```

Comment:
You have a unit test failure in this test.

Reply:
Thanks! I forgot I modified that